examples/duet/vae/AutoEncoder_Syft_Data_Owner.ipynb

```diff
@@ -1,8 +1,142 @@
 {
```

Comment:
<p>For final committed version lets remove loopback=True here and the logging line.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMined/PySyft/pull/5299/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
Done